http-poja/build.gradle

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright © 2024 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+plugins {
+    id("io.micronaut.build.internal.servlet.implementation")
+}
+
+dependencies {
+    api project(":micronaut-servlet-core")
+    implementation("com.athaydes.rawhttp:rawhttp-core:2.4.1")
+    implementation("com.athaydes.rawhttp:rawhttp-cookies:0.2.1")
```

Comment:
we need to move these to the version catalog and later replace them with something more mature